test/active_reporting/report_test.rb

```diff
@@ -33,12 +33,28 @@ def test_report_runs_with_an_aggregate_other_than_count
   end
 
   def test_report_runs_with_a_date_grouping
-    if ENV['DB'] == 'pg'
-      metric = ActiveReporting::Metric.new(:a_metric, fact_model: UserFactModel, dimensions: [{created_at: :month}])
-      report = ActiveReporting::Report.new(metric)
-      data = report.run
-      assert data.all? { |r| r.key?('created_at_month') }
-      assert data.size == 5
+    db = ENV['DB']
+    if db == 'pg' || db == 'mysql'
```

Comment:
This + the function_adapter line can be turned into a little hash I think, let me know what you think:
```ruby
adapters = {
  pg: ActiveReporting::FunctionAdapters::PostgreSql,
  mysql: ActiveReporting::FunctionAdapters::Mysql
}
db = ENV['DB']
if function_adapter = adapters[db]
  # perform operations on the adapter that you got back, keyed by db
end
```

Might want a better name for the mappings hash though